ldap-api/src/test/java/ca/bc/gov/hlth/ldapapi/service/LdapServiceTest.java

```diff
@@ -28,31 +28,28 @@ public static void initialSetup() {
 
     @Test
     public void testCreateReturnMessage_authenticatedFalse() {
-        User returnedUser = ldapService.createReturnMessage(userName, false, true, true,0,3, attributesWithRole);
+        User returnedUser = ldapService.createReturnMessage(userName, false, true, 0,3, attributesWithRole);
```

Comment:
As minor as a nitpick as possible but I just noticed a missing space between the "0,3" which looks to be the case throughout this test file.